src/main/docker/scripts/start.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Copyright 2022 Micro Focus or one of its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+export JAVA_HOME=$JRE_HOME
+chown -R opensearch:opensearch /usr/share/opensearch
+exec chroot --userspec=1000 / /usr/share/opensearch/opensearch-docker-entrypoint.sh "opensearch"
```

Comment:
Why do we have to use `chroot` to run this?

Reply:
Followed this example https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/main/elasticsearch/docker/build/elasticsearch/bin/docker-entrypoint.sh#L7
I tried setting RUNAS_USER to start OpenSearch as a non-root user, but see permission errors when building classification worker.